media/js/firefox/new/scene1-fx-experiment.js

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function(Mozilla) {
+    'use strict';
+
+    var client = Mozilla.Client;
```

Comment:
I'll refactor the way you suggested. Unless you think we should push back on the FxA status requirement all together?

Reply:
For the purposes of the experiment, it may just be fine to show the form anyway? Up to you what you think is best here. Could the form just be hidden if already signed in?

Reply:
I had 100% of this logic without FxA working and I threw it out when I realized I missed that requirement... I wish I kept it 😢